README.md

```diff
@@ -39,31 +39,33 @@ module "workmail-records" {
 
-| variable         | type   | description                                         | default                              |
-|------------------|--------|-----------------------------------------------------|--------------------------------------|
-| zone_id          | string | ID of the DNS Zone to store Records in              |                                      |
-| record_ttl       | string | TTL for all DNS records                             | `300`                                |
-| workmail_zone    | string | AWS Zone of the WorkMail Organization               | `us-east-1` |                        |
-| mx_priority      | string | MX Priority                                         | `10`                                 |
-| ownership_record | string | Ownership TXT Record                                |                                      |
-| dkim_record_1    | string | DKIM CNAME Record 1                                 |                                      |
-| dkim_record_2    | string | DKIM CNAME Record 2                                 |                                      |
-| dkim_record_3    | string | DKIM CNAME Record 3                                 |                                      |
-| spf_record       | string | SPF TXT Record                                      | `v=spf1 include:amazonses.com ~all`  |
-| dmarc_record     | string | DMARC TXT Record                                    | `v=DMARC1;p=quarantine;pct=100;fo=1` |
+| variable         | type   | description                                         | default                               |
+|------------------|--------|-----------------------------------------------------|---------------------------------------|
+| zone_id          | string | ID of the DNS Zone to store Records in              |                                       |
+| record_ttl       | string | TTL for all DNS records                             | `300`                                 |
+| workmail_zone    | string | AWS Zone of the WorkMail Organization               | `us-east-1` |                         |
+| mx_priority      | string | MX Priority                                         | `10`                                  |
+| ownership_record | string | Ownership TXT Record                                |                                       |
+| dkim_record_1    | string | DKIM CNAME Record 1                                 |                                       |
+| dkim_record_2    | string | DKIM CNAME Record 2                                 |                                       |
+| dkim_record_3    | string | DKIM CNAME Record 3                                 |                                       |
+| spf_record       | string | SPF TXT Record                                      | `v=spf1 include:amazonses.com ~all;`  |
+| apex_txt_apped   | string | additional Zone Apex TXT Record data                |                                       |
```

Comment:
`additional` is it lowercase intentionally?

Reply:
In this case, yes